crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -486,6 +486,23 @@ impl CipherView {
         }
     }
 
+    /// Extract login details for risk evaluation (login ciphers only).
+    ///
+    /// Returns `Some(CipherLoginDetails)` if this is a login cipher with a password,
+    /// otherwise returns `None`.
+    pub fn to_login_details(&self) -> Option<crate::cipher::cipher_risk::CipherLoginDetails> {
+        if let Some(login) = &self.login {
+            if let Some(password) = &login.password {
+                return Some(crate::cipher::cipher_risk::CipherLoginDetails {
+                    id: self.id,
+                    password: password.clone(),
+                    username: login.username.clone(),
+                });
+            }
+        }
+        None
```

Comment:
suggestion: Rather than the nested `if let Some()` blocks, you can use the `?` operator on Option to short-circuit these:

```suggestion
        let login = self.login.as_ref()?;
        Some(crate::cipher::cipher_risk::CipherLoginDetails {
            id: self.id,
            password: login.password.clone()?,
            username: login.username.clone(),
        })
```

Reply:
Thanks for the suggestion! [1ea9850](https://github.com/bitwarden/sdk-internal/pull/499/commits/1ea98508fd1069a84e047325bab6f67e6c8c2ef5)